concordium-sdk/src/main/java/com/concordium/sdk/responses/nodeinfo/NodeInfo.java

```diff
@@ -30,37 +32,89 @@ public class NodeInfo {
     private final ZonedDateTime localTime;
 
     /**
-     * Peer Details.
+     * Type of Peer. Bootstrapper Or Node.
      */
-    private final PeerDetails peerDetails;
+    private final PeerType peerType;
+
+    /**
+     * Type of Peer Consensus Status.
+     */
+    private final ConsensusState consensusState;
+
+    /**
+     * Type of Peer Consensus Status in Baking Committee.
+     */
+    private final BakingStatus bakingStatus;
+
+    /**
+     * Committee details. This will only be populated if {@link NodeInfo#bakingStatus}
+     * is {@link BakingStatus#ACTIVE_IN_COMMITTEE}.
+     */
+    private final BakingCommitteeDetails committeeDetails;
```

Comment:
@EmilLa1  - should we keep this optional ?

Reply:
Yes I think that was a good idea.